pom.xml

```diff
@@ -135,7 +135,7 @@
         <mongodb-driver.version>3.12.1</mongodb-driver.version>
         <mongojack.version>2.10.1</mongojack.version>
         <natty.version>0.13</natty.version>
-        <netty.version>4.1.52.Final</netty.version>
+        <netty.version>4.1.59.Final</netty.version>
         <netty-tcnative-boringssl-static.version>2.0.34.Final</netty-tcnative-boringssl-static.version>
```

Comment:
Would you mind als updating netty-tcnative to `2.0.36`? 
We've had problems with that in the past, so it's a good practice to update them together

Reply:
For reference: https://github.com/netty/netty-tcnative/releases

Reply:
Thanks for the suggestion. Done.